deck.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'model'
+require 'pry'
+
+class Deck
+  attr_reader :flashcards_from_file, :flashcards, :definition
+  attr_accessor :term
+
+  def initialize(flashcards_from_file)
+    @flashcards = []
+    flashcards_from_file.each_with_index do |element, index|
+      if index % 2 == 0
+        self.flashcards << Card.new({definition: element})
+      else
+        self.flashcards.last.term = element
+      end
+    end
```

Comment:
As mentioned above, this work of building flashcards from the file would be totally suited to its own class or module as a "factory" to build card objects.